msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/msalonly/basic/TestCase497038.java

```diff
@@ -50,6 +51,7 @@
 // https://identitydivision.visualstudio.com/DefaultCollection/DevEx/_workitems/edit/497038
 @RetryOnFailure
 @RunOnAPI29Minus("Consent Page")
+@Ignore("This one is failing because Azure Sample is 'disabled'")
```

Comment:
What do you mean by this?

Reply:
I'll reword this, this test has been changed to no longer use azure sample. And is ran manually because chrome ui is inconsistent.

Reply:
I mean I was confused with the "Azure Sample is disabled" part. not sure what that mean. 

Is Azure Sample itself broken? (even with manual test?, or is it just broken with automation?)

Reply:
The modified test seems to be using Msal Test app, so i'm not blocking on this.